### ISSUE TITLE
Unable to reproduce the results of the paper.

### ISSUE THREAD

**Issue by wyclike** [yyDing1/GNER#5]
Hello! Can you provide the inference results in a jsonl file on the CONLL2003 test set? I am unable to replicate the results you achieved on this dataset, and there is a difference of 20 points.

**Comment by wyclike** [comment]
i use your code and the model of  "GNER-LLaMA-7B'

**Comment by wyclike** [comment]
could u give the ‘llama7b-supervised.jsonl’？

**Comment by yyDing1** [comment]
Thank you for your attention!

The GNER-LLaMA-7B is the model after task adaptation (Stage 1). The CoNLL2003 Test results reported in the paper are based on the GNER-LLaMA-7B after supervised fine-tuning (Stage 2).

We provide the generation results for quick reproductions [here](https://drive.google.com/drive/folders/1kg7YDRk8jK4_Bo19jJpZtdAQMBoucppW), where xxx-stage2 refers to the supervised fine-tuning results. For Llama-7B, you can use the publicly available dataset and our training code to reproduce the results.

**Comment by wyclike** [comment]
Yeah, I knew. But there is no ‘llama7b-supervised.jsonl’ in the Google Drive folder; there are only task adaptation results.

**Comment by wyclike** [comment]
or could you please provide the post-supervised checkpoint for the llama7b  ? Thank you!

**Comment by yyDing1** [comment]
[predictions.jsonl.zip](https://github.com/user-attachments/files/17117564/predictions.jsonl.zip)

**Comment by wyclike** [comment]
Thanks！
